src/core/core.tooltip.js

```diff
@@ -95,7 +95,8 @@ module.exports = function(Chart) {
 			// Args are: (tooltipItems, data)
 			beforeFooter: helpers.noop,
 			footer: helpers.noop,
-			afterFooter: helpers.noop
+			afterFooter: helpers.noop,
+			textLabelColor: helpers.noop
```

Comment:
Also, since this is defaulting to `helpers.noop` what is the default colour of the text? I feel like we should provide a default function that has the old behaviour